tests/integration_tests/performance/test_snapshot_restore_performance.py

```diff
@@ -209,10 +209,10 @@ def test_snapshot_scaling_vcpus(
     st_core.run_exercise()
 
 
-# exponent=8 takes around 400s seconds
+# mem_exponent=7 takes around 100s
 @pytest.mark.nonci
 @pytest.mark.timeout(10 * 60)
-@pytest.mark.parametrize("mem_exponent", range(1, 9))
+@pytest.mark.parametrize("mem_exponent", range(1, 8))
```

Comment:
This should also remove the associated entries from the baseline files (unless that happens automatically the next time we collect?)

Reply:
hmm right, I think it won't happen during baselines.